erpnext/stock/stock_ledger.py

```diff
@@ -1464,8 +1464,6 @@ def get_valuation_rate(
 	if not company:
 		company = frappe.get_cached_value("Warehouse", warehouse, "company")
 
-	last_valuation_rate = None
-
```

Comment:
Removing this entirely might be confusing. Now we are kinda relying on leaky nature of walrus 😁